ksqldb-rest-app/src/main/java/io/confluent/ksql/logging/query/QueryAnonymizingRewritePolicy.java

```diff
@@ -31,7 +31,7 @@ public QueryAnonymizingRewritePolicy(final KsqlConfig config) {
     final String clusterNamespace =
         config.getString(KsqlConfig.KSQL_QUERYANONYMIZER_CLUSTER_NAMESPACE);
     this.namespace =
-        clusterNamespace.isEmpty()
+        clusterNamespace == null || clusterNamespace.isEmpty()
```

Comment:
Why don't we just change the default value of `KSQL_QUERYANONYMIZER_CLUSTER_NAMESPACE` to be the empty string?

Reply:
I think it's better to make the default empty string, I'll modify this PR. I had some behaviour that depended on clusterNamespace being null but that went away (had `Optional.ofNullable` and then figured that log4js property loader does it for me)

Reply:
Actually nevermind, this is better because the map can return null for other reasons than default being null. Going to merge this one in